tensorflow/tensorflow-0.10.0rc0.patch

```diff
@@ -0,0 +1,25 @@
+diff -ruN tensorflow-master/tensorflow/cc/BUILD tensorflow-master-patch/tensorflow/cc/BUILD
+--- tensorflow-master/tensorflow/cc/BUILD	2016-02-06 10:08:02.000000000 +0900
++++ tensorflow-master-patch/tensorflow/cc/BUILD	2016-02-06 17:42:56.446230724 +0900
+@@ -75,6 +75,21 @@
+     ],
+ )
+
++cc_binary(
++    name = "libtensorflow.so",
```

Comment:
There is already a [target for the c library](https://github.com/tensorflow/tensorflow/blob/2a10e469d70827ede21c0bb5d0a46aac97404d4c/tensorflow/BUILD#L177). Why do we need to add a new one ?